content/docs/command-reference/plots/index.md

```diff
@@ -96,7 +87,44 @@ names in the `train` array below:
 }
 ```
 
-## Plot templates (data series only)
+## Definining a plot
+
+In order to create visualizations, users need to provide the data and
+(optionally) configuration that will help customize the plot. DVC provides two
+ways to configure visualizations. Users can mark specific <abbr>stage</abbr>
+<abbr>outputs</abbr> as plot or define plot configuration inside `dvc.yaml`
+under `plots` key.
+
+## Plots definitions
```

Comment:
Suggestion: Can the headers and descriptions better explain the two different ways to define plots? Maybe introduce some terminology like "Standalone Plots" and "Stage Plots"?

Reply:
Also, should both this and the next header be subheadings of "Defining a plot"?